tests/lib/src/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set (TARGET_NAME "common")
```

Comment:
please, find more suitable name because it will be placed on file system as library (e.g. tests_lib etc.)